aws-lc-rs/tests/ecdsa_tests.rs

```diff
@@ -177,12 +181,15 @@ fn test_signature_ecdsa_verify_asn1(data_file: test::File) {
             ("P-384", "SHA3-384") => &signature::ECDSA_P384_SHA3_384_ASN1,
             ("P-521", "SHA512") => &signature::ECDSA_P521_SHA512_ASN1,
             ("P-521", "SHA3-512") => &signature::ECDSA_P521_SHA3_512_ASN1,
+            ("secp256k1", "SHA256") => &signature::ECDSA_P256K1_SHA256_ASN1,
+            ("secp256k1", "SHA3-256") => &signature::ECDSA_P256K1_SHA3_256_ASN1,
             _ => {
                 panic!("Unsupported curve+digest: {curve_name}+{digest_name}");
             }
         };
 
         let actual_result = signature::UnparsedPublicKey::new(alg, &public_key).verify(&msg, &sig);
+        println!("{curve_name}: is_valid={is_valid}, actual_result={actual_result:?}");
```

Comment:
I might be misremembering, but doesn't the test print out the results already if `assert_eq` below fails?

Reply:
No it doesn't print out the results, but I didn't mean to leave that in.